doc/source/release-notes/Percona-Server-5.6.45-86.1.rst

```diff
@@ -0,0 +1,51 @@
+.. rn:: 5.6.45-86.1
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date| (Downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-5.6/LATEST/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+This release merges changes of `MySQL 5.6.45
+<https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-45.html>`_, including
+all the bug fixes in it. Percona Server for MySQL 5.6.45-86.1 is now the current
+GA release in the 5.6 series. All of Percona’s software is open-source and free.
+
+Bugs Fixed
+================================================================================
+
+- An option has been added to conditionally enable or disable hot backup library trace information dumped to stdout. Bug fixed :psbug:`4850`
```

Comment:
Not correct.  Please use the same description used in the 8.0 release notes for the same issue:
The TokuDB hot backup library continually dumps TRACE information to the server error log. The user cannot enable or disable the dump of this information. Bug fixed #4850.

Reply:
updated.